PLUB/Sources/Views/Home/MainPage/BoardViewController.swift

```diff
@@ -86,52 +226,55 @@ extension BoardViewController: UIScrollViewDelegate {
 
 extension BoardViewController: UICollectionViewDelegate, UICollectionViewDataSource {
   func numberOfSections(in collectionView: UICollectionView) -> Int {
-    return 1
+    switch type {
+    case .clipboard:
+      return BoardViewType.allCases.count
+    case .noClipboard:
+      return BoardViewType.allCases.count - 1
+    }
   }
   
   func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-    return 10
-  }
-  
-  func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-    let cell = collectionView.dequeueReusableCell(withReuseIdentifier: BoardCollectionViewCell.identifier, for: indexPath) as? BoardCollectionViewCell ?? BoardCollectionViewCell()
-    cell.configure(with: BoardModel(author: "개나리", authorProfileImageLink: nil, date: .now, likeCount: 3, commentCount: 5, title: "게시판 제목", imageLink: nil, content: nil))
-    return cell
-  }
-  
-  func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
-    if indexPath.section == 0 {
+    let section = BoardViewType.allCases[section]
+    
+    switch section {
+    case .clipboard:
       switch type {
       case .clipboard:
-        let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: BoardCollectionHeaderView.identifier, for: indexPath) as? BoardCollectionHeaderView ?? BoardCollectionHeaderView()
-        return header
+        return clipboardModel.count // 현재 클립보드에 올려져있는 글 갯수
       case .noClipboard:
-        return UICollectionReusableView()
+        return 0
       }
+    case .normalSystem:
+      return 10
     }
-    return UICollectionReusableView()
-  }
-  
-  func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
     
   }
   
-  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
-    switch type {
+  func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+    let section = BoardViewType.allCases[indexPath.section]
+    switch section {
     case .clipboard:
-      return CGSize(width: collectionView.frame.width, height: 260 + 22)
-    case .noClipboard:
-      return .zero
+      let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ClipboardCollectionViewCell.identifier, for: indexPath) as? ClipboardCollectionViewCell ?? ClipboardCollectionViewCell()
+      cell.configureUI(with: clipboardModel.map({ model in
+        return ClipboardCollectionViewCellModel(type: model.type, contentImageString: model.feedImageURL, contentText: model.content)
+      })[indexPath.row]
+      )
```

Comment:
하나로 묶어 쓰는 것 보다는 한 번 나눠서 쓰는 게 가독성에 더 좋을 거 같아요. :)
```suggestion
      let model = clipboardModel.map { ClipboardCollectionViewCellModel(type: $0.type, contentImageString: $0.feedImageURL, contentText: $0.content) }
      cell.configureUI(with: model[indexPath.row])
```

아니면, 전부 map할 필요도 없으니 성능까지 고려한다면 아래처럼 처리하면 될 것 같네요.

```suggestion
      let model = ClipboardCollectionViewCellModel(
        type:               clipboardModel[indexPath.row].type,
        contentImageString: clipboardModel[indexPath.row].feedImageURL,
        contentText:        clipboardModel[indexPath.row].content
      )
      cell.configureUI(with: model)
```

Reply:
이 부분 API 데이터받아온게 아니라서 임의로 작업해둔건데 수정하도록 하겠습니다 👍